pontoon/base/urls.py

```diff
@@ -60,7 +60,7 @@
         name='pontoon.translate'),
 
     # List contributors
-    url(r'^contributors/$',
+    url(r'^contributors/(?P<period>overall|12months|6months|3months|lastmonth)$',
```

Comment:
Let's use alltime instead of overall to match the menu entry.

Also, I think we should keep the old link (/contributors) to avoid breaking any (external) links. It should be redirected to /contributors/alltime